components/buttongroup/events.md

```diff
@@ -100,7 +102,7 @@ The `OnClick` event fires when the user clicks or taps the button. You can use i
 
         result = sender + DateTime.Now.Millisecond;
     }
-}
```

Comment:
Should this go back in?

Reply:
Done with [28ff9e9](https://github.com/telerik/blazor-docs/pull/232/commits/28ff9e9acc51133b9b66d33b7b8673203d7e5a1c)